1223-curator_worklist/358-curator_opens_a_registration_from_a_doi_request.feature

```diff
@@ -9,3 +9,37 @@ Feature: Curator opens a Registration from a DOI Request
         Then they see the Landing Page for the DOI Request's Registration
         And the Create DOI button is enabled
         And the Decline DOI button is enabled
+
+    @512
+    Scenario: A Curator approves a DOI request
+        Given that a Curator opens a Publication from a DOI Request Worklist Item
+        When they click Create DOI
+        Then they see the Public Page for Publication
+        And the Publication has a DOI Link
```

Comment:
Vi har prøvd å være konsekvent på bruken av Registration i stedet for Publication i nyere scenario
```suggestion
        Then they see the Public Page for Registration
        And the Registration has a DOI Link
```